DependencyInjection/Security/Factory/LtiLaunchRequestSecurityFactory.php

```diff
@@ -45,10 +45,10 @@ public function getKey(): string
 
     public function create(
         ContainerBuilder $container,
-        string $id,
-        array $config,
-        string $userProvider,
-        ?string $defaultEntryPoint
+        $id,
```

Comment:
can I ask you to add a TODO as a comment to revert this, the day we will move all our apps to SF > 4.2 ?